docs/feature-store/sources-targets.md

```diff
@@ -199,6 +199,26 @@ For example:
 Disable partitioning with:
 - `ParquetTarget(partitioned=False)`
 
+## Snowflake target
+
+```python
+profile = DatastoreProfileSnowflakeTarget(name="profile-name",bootstrap_servers="localhost", topic="topic_name")
```

Comment:
snowflake does not have datastore profile:)

Reply:
deleted